marklogic-data-hub/src/main/resources/ml-modules-final/options/exp-default.xml

```diff
@@ -0,0 +1,43 @@
+<options xmlns="http://marklogic.com/appservices/search">
+  <constraint name="Collection">
+    <collection/>
+  </constraint>
+  <constraint name="createdByJob">
+    <value>
+      <field name="datahubCreatedByJob"/>
+    </value>
+  </constraint>
+  <constraint name="createdByStep">
+    <value>
+      <field name="datahubCreatedByStep"/>
+    </value>
+  </constraint>
+  <constraint name="createdByJobRange">
+    <range>
```

Comment:
Note that range defaults to facet=true. Unless you need a facet on job ID - seems unlikely? - you should add facet=false.

Reply:
@rjrudin This is a requirement for the explorer to enable faceting on JobId's. For datahubCreatedOn we are using a date-picker. So I am setting the facet="false" for the createdOn range constraint

Reply:
@rjrudin Removed faceting on createByJobRange but retained the constraint and created a new commit